python/requirements/requirements_tune.txt

```diff
@@ -34,7 +34,7 @@ torchvision==0.8.1;sys_platform=="darwin"
 
 # On non-OSX machines only install CPU version of torch and torchvision
 -f https://download.pytorch.org/whl/torch_stable.html
-torch==1.7.0+cpu;sys_platform!="darwin"
+torch==1.7.1;sys_platform!="darwin"
```

Comment:
well we still want the cpu tag. (but can live without it)

Reply:
You're right. Darn I guess Dependabot doesn't automatically handle this 😕